src/main/resources/anniv5Resources/localization/eng/doppelpack/Keywordstrings.json

```
@@ -0,0 +1,17 @@
+[
+  {
+    "PROPER_NAME": "Summon",
+    "NAMES": [
+      "summon"
+    ],
+    "DESCRIPTION": "Summon a #yDoppel and put it into the first slot. If you have no empty slots, your first #yDoppel is automatically #yEvoked to make room."
```

```suggestion
    "DESCRIPTION": "#yChannel a #yDoppel and put it into the first slot. If you have no empty slots, your first #yDoppel is automatically #yEvoked to make room."
```
Might be good to use the channel keyword here to make sure players understand Doppels are basically orbs and occupy the same space.

> put it into the first slot
Should this be the first **free** slot?